sources/packages/backend/libs/integrations/src/services/disbursement-schedule-errors/disbursement-schedule-errors.service.ts

```diff
@@ -50,4 +51,24 @@ export class DisbursementScheduleErrorsService extends RecordDataModelService<Di
       .orIgnore()
       .execute();
   }
+
+  /**
+   * Checks if there any full time disbursement feedback errors.
+   * @param disbursementScheduleId disbursement schedule id.
+   * @returns true, if there any full time disbursement feedback
```

Comment:
Please adjust to "true, if there are any full-time disbursement feedback errors".

Reply:
@ann-aot to be clear the problem here was "there any".